shared/ui/package-lock.json

```diff
@@ -0,0 +1,590 @@
+{
```

Comment:
Please remove

Reply:
Reminder for me: also push PR to remove this pkg from lerna bootstrap